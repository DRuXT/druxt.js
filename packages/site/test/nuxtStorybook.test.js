import DruxtEntityStorybook from '../src/nuxtStorybook'

jest.mock('axios')

const mock = {
  addTemplate: jest.fn(),
  options: {
    buildDir: '',
    druxt: {
      baseUrl: 'https://demo-api.druxtjs.org'
    }
  }
}

test('Nuxt Storybook', async () => {
  const stories = []
  await DruxtEntityStorybook.call(mock, { options: mock.options.druxt, stories })
  expect(stories.length).toBe(4)
})
