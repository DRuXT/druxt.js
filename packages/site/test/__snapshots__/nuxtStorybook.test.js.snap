// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nuxt Storybook 1`] = `
Array [
  "/stories/druxt-site.stories.js",
  "/stories/druxt-site.*.stories.js",
]
`;
